ssr/render.js

```diff
@@ -242,12 +231,16 @@ export default function render(
     // The snippet is always the same, if it's present, but the ID varies
     // See LUX settings here https://speedcurve.com/mozilla-add-ons/mdn/settings/lux/
     const speedcurveJS = getSpeedcurveJS();
-    $("<script>").text(`\n${speedcurveJS}\n`).appendTo($("head"));
+    $("<script>").text(speedcurveJS).appendTo($("head"));
     $(
       `<script src="https://cdn.speedcurve.com/js/lux.js?id=${SPEEDCURVE_LUX_ID}" async defer crossorigin="anonymous"></script>`
     ).appendTo($("head"));
   }
 
+  $(
+    '<script defer src="http://polyfill.io/v3/polyfill.min.js?features=Array.prototype.flat%2CArray.prototype.includes"></script>'
+  ).appendTo($("head"));
```

Comment:
But now every single page load will require fetching this. It can be cached by a `vary` header so it'll be pretty fast for every individual browser. But it still bothers me a bit since this is a `defer` load it means it won't be able to *execute* any loaded `.js` after this. So if `polyfill.io` is just having a bad case of the Mondays suddenly, all the other `.js` execution has to wait. 

Why can't we just use a `nonce`? You can just generate any random string based the seed of what was there in `index.html` and then stick that in. Now, you can keep the `inline` script just fine and still have a valid CSP. That way only a tiny fraction of clients have to every add a network lookup to `polyfill.io` as a blocker for the rest of the JS execution.

Reply:
true good point. How about we make it `async`? I guess we'd still have the problem that it might load too slow and browsers without `.flat/.includes` would error if such a call is reached before the polyfill is finally resolved.